auth/api/iam/api.go

```diff
@@ -153,3 +184,19 @@ func (r Wrapper) HandleAuthorizeRequest(ctx context.Context, request HandleAutho
 		//Description: "missing or invalid parameters",
 	}
 }
+
+func createSession(params map[string]string, ownDID did.DID) *Session {
+	session := &Session{
```

Comment:
I expect different authorizations flows to use different parameters. Maybe just store the entire request? (or defer to different issue/PR)

Reply:
Yeah, I'd refactor that when we introduce the next flow requiring these parameters. I think we should only store specified parameters, otherwise we're suspect to resource-exhaustion attacks? (attackers making up parameter names)